src/main/java/org/craftercms/studio/api/v2/service/cluster/ClusterManagementService.java

```diff
@@ -31,11 +33,26 @@ public interface ClusterManagementService {
     List<ClusterMember> getAllMemebers();
 
 
+    /**
+     * Get member by local address
+     * @param localAddress
+     * @return cluster member
+     */
+    @HasPermission(type = DefaultPermission.class, action = "read_cluster")
```

Comment:
Please switch this from authentication to using the ManagementToken as we did for Monitoring.

Reply:
Switched to ManagementToken.